google/cloud/firestore_v1/document.py

```diff
@@ -389,29 +388,34 @@ def get(
                 :attr:`create_time` attributes will all be ``None`` and
                 its :attr:`exists` attribute will be ``False``.
         """
-        request, kwargs = self._prep_get(field_paths, transaction, retry, timeout)
+        request, kwargs = self._prep_batch_get(field_paths, transaction, retry, timeout)
 
         firestore_api = self._client._firestore_api
-        try:
-            document_pb = firestore_api.get_document(
-                request=request, metadata=self._client._rpc_metadata, **kwargs,
-            )
-        except exceptions.NotFound:
-            data = None
-            exists = False
-            create_time = None
-            update_time = None
-        else:
-            data = _helpers.decode_dict(document_pb.fields, self._client)
+        read_time: Timestamp = None
+        data = None
+        exists = False
+        create_time = None
+        update_time = None
+
+        response_stream = firestore_api.batch_get_documents(
+            request=request, metadata=self._client._rpc_metadata, **kwargs,
+        )
+        response: List[BatchGetDocumentsResponse] = list(response_stream)
+
+        batch_pb: BatchGetDocumentsResponse = response[0]
```

Comment:
nit: this used to actually be a pb type, do you think renaming to response or batch_response would be better?

Reply:
Updated to just `batch`.